### PR TITLE
Remove ssl certificate checking from webdriver

### PR DESCRIPTION
**Pull request by davidkelliott**
The CWA UAT environment has a self signed certificate, which causes a
browser error when running the tests.  This change removes the ssl
checking (there is no option to add exceptions that I could find) to
allow the tests to run.

Also updated the cwa_env.yml file with the correct values for UAT.

### CODE REVIEW

**Comment by sldblog** [comment]
> UAT environment has a self-signed certificate

:neckbeard: The landing page (Portal) has a correct certificate on both `tst` and `uat` environments. Is this referring to CWA?

**Comment by davidkelliott** [comment]
> > UAT environment has a self-signed certificate
> 
> :neckbeard: The landing page (Portal) has a correct certificate on both `tst` and `uat` environments. Is this referring to CWA?

Yes sorry I should have been more specific, that's correct.

File: features/support/env.rb [hunk 1]
```diff
@@ -2,6 +2,12 @@
 require 'rspec/expectations'
 require 'byebug'
 require 'awesome_print'
+require 'selenium-webdriver'
+
+Capybara.register_driver :selenium do |app|
+  capabilities = Selenium::WebDriver::Remote::W3C::Capabilities.firefox(accept_insecure_certs: true)
```
Comment (sldblog) [inline features/support/env.rb]: According to https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/Remote/W3C/Capabilities, this is a private API. Could we use public API alternatives, please? (I _think_ that may be `Selenium::WebDriver::Firefox::Options` or similar).

Comment (davidkelliott) [reply features/support/env.rb]: > According to https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/Remote/W3C/Capabilities, this is a private API. Could we use public API alternatives, please? (I _think_ that may be `Selenium::WebDriver::Firefox::Options` or similar).

I've been looking but can't see how to do this, I'm in the office tomorrow, maybe we could pair on this if you have time?

Comment (davidkelliott) [reply features/support/env.rb]: I still haven't found a way around this, this is also the method given in the selenium Ruby bindings docs - 
https://github.com/SeleniumHQ/selenium/wiki/Ruby-Bindings#ssl-certificates

Comment (sldblog) [reply features/support/env.rb]: Fair enough 👍 
(And sorry, this slipped off my list of "things I need to get back to" 😞)

Comment (davidkelliott) [reply features/support/env.rb]: No worries, thanks :)

**Pull request merged**